### DISCUSSION TITLE
What Device I should Select

### DISCUSSION THREAD

**Discussion by dmoraca** [make-all/tuya-local#2505] [General]
Hello everyone,

I’m trying to add a Wi-Fi switch with dimming capabilities to Home Assistant using Tuya Local. My setup involves pot lights, which are not RGB-enabled, so I don’t need any RGB controls.

Problem:
I’m unsure which settings to select for this device in Tuya Local, given that it’s a simple dimmer without RGB functionality. I’d like to control the on/off state and adjust the brightness level, but color isn’t necessary.

Questions:

What device configuration should I choose in Tuya Local for a dimmer-only setup?
Are there specific settings to skip or adjust since the lights are not RGB?
Any tips for ensuring smooth integration and avoiding unnecessary RGB controls in the UI?
Thank you in advance for any advice or guidance!

![image](https://github.com/user-attachments/assets/14502790-e7f9-41ca-a879-364f88819827)

Please help
Thank you

**Comment by make-all** [comment]
Without knowing more information about your specific device, it is impossible to say which config might work (by ignoring any capabilities your device does not have).

**Reply by HellixFix** [reply]
I see my Moes dimmer in custom components but unable to select here. How to choose it? It is not in the list....

**Comment by dmoraca** [comment]
Because I cannot see proper device type when using Tuya Local I have decided to purchase ZBT-1 SkyConnect - Zigbee. Hopefully I will be able to connect these switches this way. Do you think this is possible?

**Reply by make-all** [reply]
You wrote above they are WiFi switches. A Zigbee stick will not help connect to WiFi switches, but it will allow you to buy Zigbee devices in future and avoid the Tuya cloud completely.

**Reply by dmoraca** [reply]
And I was hopeful I will trick them, but they tricked me hahaha. Thanks for that

**Comment by dmoraca** [comment]
I was able to select "gosund_SW2_dimmer_switch" that fixed my problem in latest update of Tuya Local
